docs/overview/trustbloc-did-method.md

```diff
@@ -57,22 +82,26 @@ Each of these files is named `[domain].json`, where `[domain]` is the URL domain
 A stakeholder's config file is a JWS, signed by the stakeholder, with the payload being a JSON object containing:
 - The stakeholder's domain
 - The stakeholder's DID (did:trustbloc)
-  - TODO: also include a did:web DID doc? Same DID, same verkeys, used for bootstrapping.
 - Stakeholder custom configuration settings
 - The stakeholder's Sidetree endpoints
 - the hash of the previous version of this config file
 
-TODO: this should probably be in a secondary file, or stored in a way that's easy to detach without risk of screwing up the payload text
-- A detached JWS signature on the file, created by the stakeholder
+```json
+{
+    "domain": "stakeholder.one",
+    "did": "did:bloc:[consortium domain]:[stakeholder one's DID]",
```

Comment:
```suggestion
    "did": "did:trustbloc:[consortium domain]:[stakeholder one's DID]",
```

Reply:
"stakeholder one's DID" is the last portion of this trustbloc DID. So we have a DID within a DID, eg.:

```
did:trustbloc:myconsortium.com:did:sidetree:1234567abcdefg;initial-values=<encoded-create-operation-request>
```

Can we drop the `did:sidetree:` portion from the embedded DID?

Reply:
This being an example, I can instead drop the `did:trustbloc:[consortium domain]:` bit, and leave it as saying it's the stakeholder's DID.

Reply:
Done.